BLUESPAWN-client/src/user/BLUESPAWN.cpp

```diff
@@ -67,6 +68,7 @@ Bluespawn::Bluespawn() {
 	MitigateV1153* v1153 = new MitigateV1153(mitigationRecord);
 	MitigateV3338* v3338 = new MitigateV3338(mitigationRecord);
 	MitigateV3340* v3340 = new MitigateV3340(mitigationRecord);
+	MitigateV3344* v3344 = new MitigateV3344(mitigationRecord);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable 'v3344' is assigned a value that is never used.](https://app.codacy.com/manual/ION28/BLUESPAWN/pullRequest?prid=4972089)